src/Servers/Kestrel/test/FunctionalTests/RequestTests.cs

```diff
@@ -500,7 +500,6 @@ public async Task AbortingTheConnectionSendsFIN()
 
         [Theory]
         [MemberData(nameof(ConnectionMiddlewareData))]
-        [QuarantinedTest("https://github.com/dotnet/aspnetcore/issues/23043")]
```

Comment:
There's no indication this issue has been addressed and it's still open. Why unquarantined it?

Reply:
The ops instructions are unclear then. Maybe we should update it. All it says is if I see a quarantined test running and passing for the past 30 days, I should unquarantine it

Reply:
Yes, it's time to re-write those.
A) The quarantine builds have some known gaps like not running on some platforms.
B) A test should not be unquarantined without some investigation and improvement.